scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1552,17 +1442,9 @@
                         (assoc-in [:pool->offers-chan pool-name] offers-chan)
                         (assoc-in [:pool->resources-atom pool-name] resources-atom))))
                 {}
-                pools')
-        {:keys [batch-size]} progress-config
```

Comment:
This removal/migration makes me a little nervous. How can we be sure that this all works?

Reply:
I'll let you be the judge of 'enough testing'. :)